frontend/src/pte-specifics/commands/player.js

```diff
@@ -25,6 +26,7 @@ const player = {
         return JSON.parse(JSON.parse(results.outputs[0]).value);
```

Comment:
Im actually thinking of using @cbisaillon's suggestion to log the json and use the logs array instead, saves having to use `JSON.parse()` twice even though its super fast anyway

Reply:
Awesome - feel free to add / remove as you see fit 🤙